Directory.Build.props

```
@@ -62,6 +62,11 @@
     <MachineIndependentBuild>true</MachineIndependentBuild>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AssemblyInformationCachePaths Condition="Exists('$(MSBuildBinPath)\SdkPrecomputedAssemblyReferences.cache')">$(MSBuildBinPath)\SDKPrecomputedAssemblyReferences.cache</AssemblyInformationCachePaths>
+    <AssemblyInformationCachePaths Condition="Exists('$(NetCoreRoot)$(NetCoreSdkVersion)\SdkPrecomputedAssemblyReferences.cache')">$(NetCoreRoot)$(NetCoreSdkVersion)\SDKPrecomputedAssemblyReferences.cache</AssemblyInformationCachePaths>
```

I looked at binlogs from a framework build and a core build, and it looks like this is the change necessary. The other line was necessary because this one always failed.

```suggestion
    <AssemblyInformationCachePaths Condition="Exists('$(NetCoreRoot)sdk\$(NetCoreSdkVersion)\SdkPrecomputedAssemblyReferences.cache')">$(NetCoreRoot)$(NetCoreSdkVersion)\SDKPrecomputedAssemblyReferences.cache</AssemblyInformationCachePaths>
```